client/src/App.js

```diff
@@ -22,7 +23,8 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/movielist" component={MovieList} />
```

Comment:
the initial url path on load is `/`, so when first starting up your app, only the SavedList is shown with no way of directing to the rest of the app without adding `/movielist` at the end of the browser url